SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -161,9 +161,11 @@
   private long saveStateEverySeconds = 60;
 
   @JsonProperty("sentry")
+  @Valid
```

Comment:
`sentryConfiguration` and `smtpConfiguration` can be `null` -- could you sanity check that a `null` value annotated with `@Valid` is legal? (i.e. we don't want it to fail validation unless it also has a `@NotNull`)